src/main/java/gameframework/gui/GameWindow.java

```diff
@@ -18,18 +18,17 @@
 	protected final ObservableValue<Integer> life;
 
 	public GameWindow(GameCanvas gameCanvas, GameConfiguration configuration,
-			ObservableValue<Integer> score, ObservableValue<Integer> life) {
-		this("Default Game", gameCanvas, configuration, score, life);
+			ObservableValue<Integer> score) {
+		this("Default Game", gameCanvas, configuration, score);
 	}
 
 	public GameWindow(String gameName, GameCanvas gameCanvas,
-			GameConfiguration configuration, ObservableValue<Integer> score,
-			ObservableValue<Integer> life) {
+			GameConfiguration configuration, ObservableValue<Integer> score) {
 		if (gameCanvas == null) {
 			throw new IllegalArgumentException("gameCanvas is null");
 		}
 		this.score = score;
-		this.life = life;
+		this.life = new ObservableValue<Integer>(configuration.getDefaultNbLives());
```

Comment:
This is not correct. You should not create multiple observable objects for the same value. That breaks the Observer design pattern. A better alternative would be to pass the `GameData` as argument and then fetch the other values from data (life, score, configuration, ...)